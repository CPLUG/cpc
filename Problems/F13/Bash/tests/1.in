9
.bashrc
.bash_profile
.bash_history
.vimrc
.dmrc
.htoprc
.mybashrc
.pinerc
.screenrc
