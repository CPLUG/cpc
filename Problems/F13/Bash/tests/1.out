.{bash{_{history,profile},rc},dmrc,htoprc,mybashrc,pinerc,screenrc,vimrc}
